test/HybridCustody_tests.cdc

```diff
@@ -914,6 +965,22 @@ pub fun addTypeToFilter(_ acct: Test.Account, _ kind: String, _ identifier: Stri
     txExecutor(filePath, [acct], [identifier], nil, nil)
 }
 
+pub fun removeAllFilterTypes(_ acct: Test.Account, _ kind: String) {
+    var filePath = ""
+    switch kind {
+        case FilterKindAllowList:
+            filePath = "filter/allow/remove_all_types.cdc"
```

Comment:
Looks like the file needs to be renamed - `remoe_all_types.cdc` -> `remove_all_types.cdc`. Small typo is causing this test case to fail